ngraph/core/src/op/variadic_split.cpp

```diff
@@ -171,15 +171,12 @@ namespace
                                  const HostTensorVector& outputs,
                                  const Node* split_node)
     {
-        int64_t axis;
         switch (axis_tensor->get_element_type())
```

Comment:
as above

Reply:
I see that variadic_split has evaluate() so binary size concern is valid. Why remove it if it was done on purpose?